GNUmakefile

```diff
@@ -104,6 +104,13 @@ ifeq (${BASE_DOCKER_IMAGE_ARM64}, )
   export BASE_DOCKER_IMAGE_ARM64
 endif
 
+# Specify the name of base image for PPC64LE
+ifeq (${BASE_DOCKER_IMAGE_PPC64LE}, )
+  BASE_DOCKER_IMAGE_PPC64LE = "ubuntu:18.04"
```

Comment:
Can we directly use `ubuntu:18.04` or do we need to use `ppc64le/ubuntu:18.04`

Reply:
@akhilerm That is a multi arch image, so it can be used directly as is.